pkg/apis/batch/validation/validation.go

```diff
@@ -237,6 +243,18 @@ func validatePodFailurePolicy(spec *batch.JobSpec, fldPath *field.Path) field.Er
 	return allErrs
 }
 
+func validatePodReplacementPolicy(spec *batch.JobSpec, onReplacePath *field.Path) field.ErrorList {
```

Comment:
```suggestion
func validatePodReplacementPolicy(spec *batch.JobSpec, fldPath *field.Path) field.ErrorList {
```
nit, I think it will be more consistent with other places.